src/enforce/rules/helpers/is_not_type/index.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import isType from '../is_type';
+
+/**
+ * Check if value type does not match any provided types
+ * Inverse of isType
+ * @see {@link isType}
+ * @see isType
+ */
+export function isNotType(value: mixed, ...types: Array<string>): boolean {
```

Comment:
As noted in your comment:
> should consider deprecating 'isNotType' helper (not actually used in source code)

There is no benefit to having it a part of the codebase at the moment, and it can be removed.

Reply:
indeed